src/model/SubSurface.hpp

```diff
@@ -263,6 +263,12 @@ namespace model {
 
     boost::optional<AirflowNetworkSurface> airflowNetworkSurface() const;
```

Comment:
Suggest adding `frameArea` and `dividerArea` methods

Reply:
* [ ] TODO:  I think that makes sense indeed.

Reply:
I added these methods as suggested. Frame area is pretty obvious but for divider area I used the number of horizontal dividers, the divider width and the width of the sub surface and similarly for vertical with the height of the sub surface. Any better ideas?

Reply:
I don't see those implementations in this PR?  Did they get moved?

Reply:
Probably just a git push missing @ggartside